source/module_elecstate/potentials/potential_new.cpp

```diff
@@ -18,6 +18,7 @@
 namespace elecstate
 {
 Potential::Potential(const ModulePW::PW_Basis* rho_basis_in,
+                     const ModulePW::PW_Basis* rho_basis_smooth_in,
```

Comment:
Please add brief introduction of the differences between rho_basis and rho_basis_smooth

Reply:
Add it in .h file